canonical/ubuntu.com
# removing the form from the /download/server/linuxone page

## Done

removing form from the /download/server/linuxone page per Jim Foley and Tom Callway request
## QA
1. go to /download/server/linuxone
2. see that there is no form and the page looks like the server
3. click the 'Download' button and see that it goes to the thank you and starts a download
